packages/jack_link/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_link_commit="41d9aa111f702e78b6fbaee9d3e06dda1db6420d"
+pkgname=jack_link
```

Comment:
My first thought was that I've never seen a pkgname with an underscore, but apparently that's perfectly fine according to the wiki, and `jack_capture` is another example doing that 👍

Reply:
Even the official repos have packages with underscores in their names.